src/app/network/upload.ts

```diff
@@ -127,15 +131,22 @@ export function uploadFile(bucketId: string, params: IUploadParams): Promise<str
   params.abortController?.signal.addEventListener('abort', onAbort);
 
   if (useMultipart) {
-    uploadPromise = facade.uploadMultipart(bucketId, params.mnemonic, file, {
-      uploadingCallback: params.progressCallback,
-      abortController: uploadAbortController,
-      parts: Math.ceil(params.filesize / partSize),
-    });
+    uploadPromise = facade.uploadMultipart(
+      bucketId,
+      params.mnemonic,
+      file,
+      {
+        uploadingCallback: params.progressCallback,
+        abortController: uploadAbortController,
+        parts: Math.ceil(params.filesize / partSize),
+      },
+      params?.continueUploadOptions,
```

Comment:
Why the continue upload is not inside the params body like the uploading callback or the abortController?